build/linux/debian/install-sysroot.ts

```diff
@@ -45,6 +45,7 @@ function getElectronVersion(): Record<string, string> {
 }
 
 function getSha(filename: fs.PathLike): string {
+	// CodeQL [SM04514] Hash logic cannot be changed due to external dependency, also the code is only used during build.
```

Comment:
Are you sure the `Code QL [*]` directives work on the line above as well? I assumed it was only on the actual warning line.

Reply:
Went be the valid examples in the [suppression guide](https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/codeql-semmle?#guidance-on-suppressions)

Reply:
Perfect!